drivers/spi/spi_slave_driver.c

```diff
@@ -57,7 +57,7 @@
  * Private Types
  ****************************************************************************/
 
-struct spislv_driver_s
+struct spislave_driver_s
```

Comment:
should we give the driver a special name? since it is most used for simple echo testing.

Reply:
At first I thought of calling it `spi_echo`, but then I changed my mind because it would be too restrictive.

It may be extended for further than just echo testing.
One may create a test application that gives real responses to commands from Master.
The driver idea is to serve just a bridge to the userspace.

Reply:
Ok. how about the prefix(spislave_->spi_sdev_)?

Reply:
The `spislave_driver_s` follows the same pattern from the `spi_driver_s` naming, considering `spislave_` as the single prefix.

I agree that `spi_sdev_` would be in sync with the prefix adopted in `slave.h`, but it is difficult to infer that `sdev` refers to Slave Device. The same with `spi_sctrlr`. On the other hand, `spislave_dev` and `spislave_ctrlr` are self explanatory, don't you think?

Reply:
Here is the convention used by i2c slave:
i2c_slave_s/i2c_slaveops_s/I2CS_XXX
How about we use the similar convention for spi slave:
spi_slave_dev_s/spi_slave_devops_s/spi_slave_ctrlr_s/spi_slave_ctrlrops_s/SPIS_DEV_XXX/SPIS_CTRLR_XXX

Reply:
I wasn't aware of the I2C Slave interface, thanks for letting me know.
This proposal is nice, I agree with the change.
I can submit this change tomorrow, or feel free to submit it.

Reply:
Sure, let's wait your PR.